### PR TITLE
BUG PM-681: When user logs out from metamask, mainnet address and reward info are still shown

### PR DESCRIPTION
**Pull request by mmvsha**
### Description
* Set mainnet address to null when user logs out from metamask

### Which Tickets does my PR fix? (Put in title too)
* Fixes bug/PM-681

### Which side effects could my PR have?
* Can't think of any

### Which Steps did I take to verify my PR?

Logged out/in with metamask like 3 times and it worked correctly (hid/showed the account, rewards info)

### CODE REVIEW

File: src/integrations/store/actions/index.js [hunk 1]
```diff
@@ -24,6 +25,15 @@ export const logoutProvider = () => async (dispatch, getState) => {
 }
 
 export const runProviderUpdate = (provider, data) => async (dispatch, getState) => {
+  if (provider.constructor.providerName === WALLET_PROVIDER.METAMASK && isFeatureEnabled('tournament')) {
```
Comment (andre-meyer) [inline src/integrations/store/actions/index.js]: @mmvsha Hmm, so when the user uses uPort, the address will remain?

Comment (andre-meyer) [reply src/integrations/store/actions/index.js]: I think this part can be removed, it should always delete the mainnet address when the wallet is locked/closed

Comment (mmvsha) [reply src/integrations/store/actions/index.js]: we don't set mainnet address for uPort users @andre-meyer 
how do you detect if the wallet is closed?

Comment (andre-meyer) [reply src/integrations/store/actions/index.js]: This was resolved :) Feel free to merge

**Pull request merged**